internal/tracing/keyboard_metrics.go

```diff
@@ -22,14 +22,19 @@ import (
 	"go.opentelemetry.io/otel/attribute"
 )
 
-func KeyboardMetrics(ctx context.Context, enabled, isDockerDesktopActive, isWatchConfigured bool) {
+func KeyboardMetrics(ctx context.Context, enabled, isDockerDesktopActive, isWatchConfigured, isDDComposeUI bool) {
 	commandAvailable := []string{}
 	if isDockerDesktopActive {
 		commandAvailable = append(commandAvailable, "gui")
 	}
 	if isWatchConfigured {
 		commandAvailable = append(commandAvailable, "watch")
 	}
+
+	if isDDComposeUI {
```

Comment:
for consistency with `isDockerDesktopActive` I would change it to `isDockerDesktopComposeUI`